tools/wake-format/emitter.cpp

```diff
@@ -83,7 +94,12 @@ wcl::doc Emitter::walk(ctx_t ctx, CSTElement node) {
   auto extra_fmt = fmt()
       .fmt_if(TOKEN_WS, fmt().next())
       .fmt_if(TOKEN_NL, fmt().next().newline())
-      .fmt_if(TOKEN_COMMENT, fmt().walk(WALK(walk_token)));
+      .fmt_if([](wcl::doc_builder& builder, ctx_t ctx, CSTElement& node){
```

Comment:
Should these two predicates be given names? Like "is_format_off_token" and "is_inert_comment"?

Actully you don't need the `is_inert_token` just `TOKEK_COMMENT` is fine. That way when you make this a match thing you can still assert if you see a bad token

Reply:
they are both gone now :)